cargo-miri/bin.rs

```diff
@@ -586,11 +587,10 @@ fn phase_cargo_miri(mut args: env::Args) {
     };
     let subcommand = match &*subcommand {
         "setup" => MiriCommand::Setup,
-        "test" | "t" | "run" | "r" => MiriCommand::Forward(subcommand),
-        // Invalid command.
+        "test" | "t" | "run" | "r" | "nextest" => MiriCommand::Forward(subcommand),
```

Comment:
Did the `// Invalid command.` comment get lost during a rebase?

Reply:
I think so yeah, whoops!